libs/langchain-google-common/src/connection.ts

```diff
@@ -159,7 +163,7 @@ export abstract class GoogleAIConnection<
   extends GoogleHostConnection<CallOptions, GoogleLLMResponse, AuthOptions>
   implements GoogleAIBaseLLMInput<AuthOptions>
 {
-  model: string;
+  modelName: string;
```

Comment:
It's fine if there's a reason since this is all undocumented but why the change?